docs/integrations/persisting-store-data.md

```diff
@@ -544,7 +544,7 @@ withStorageDOMEvents(useBoundStore)
 
 ### How do I use with TypeScript?
 
-Basic typescript usage doesn't require anything special except for writing `create<State>()(...)` instead of `create(...)`.
+Basic typescript usage require define middleware and state like below.
```

Comment:
Thanks @moonrailgun .  I think there is a word or two missing in this updated sentence. 

How about something along this lines: 

"To use the persist middleware with TypeScript, you should describe both the shape of your store as well as each middleware being introduced. `create` accepts two generic parameters. Refer to the example below.

Reply:
well. your word is more detail and nice, look good to me and i follow it

Reply:
and a detail is `generic` is not same with `generics`. i think `generics` is more fit in this case